src/aapt2.tpnitems

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(TpnIncludeExternalDependencies)' == 'True' ">
+    <ThirdPartyNotice Include="google/aapt2">
+      <LicenseText>
```

Comment:
Instead of using `<LicenseText/>`, please use:

```xml
<LicenseFile>$(MSBuildThisFileDirectory)..\build-tools\license-data\Apache-2.0.txt</LicenseFile
```

We don't need *N* different copies of the same license text.